packages/react/scripts/generatePrimitivesCatalog.ts

```diff
@@ -168,6 +168,28 @@ for (const [componentName, [node]] of source.getExportedDeclarations()) {
   }
 }
 
-// Generates dist/primitives.json file
-const outputPath = path.resolve(__dirname, '..', 'dist', 'primitives.json');
-fs.writeFileSync(outputPath, JSON.stringify(catalog, null, 2));
+/**
+ * Generate the JSON string of the PrimitiveCatalog
+ * this is being exported under the /primitives.json subpath and can be used as
+ * import PrimitiveCatalog from '@aws-amplify/ui-react/primitives.json'
+ */
+const jsonString = JSON.stringify(catalog, null, 2);
+
+/**
+ * Generate the es module of the PrimitiveCatalog
+ * this is being exported under the /internal/primitive subpath and can be used as
+ * import { PrimitiveCatalog } from '@aws-amplify/ui-react/internal/primitive'
+ */
+const exportString = `export const PrimitiveCatalog = ${jsonString};
+export default PrimitiveCatalog;`;
```

Comment:
Should we use a named export to stay consistent with the rest of the repo?

```suggestion
export { PrimitiveCatalog }`;
```